lisa/features/network_interface.py

```diff
@@ -41,6 +41,9 @@ def reload_module(self) -> None:
     def get_nic_count(self, is_sriov_enabled: bool = True) -> int:
         raise NotImplementedError
 
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self.origin_extra_nics: Dict[str, int] = {}
```

Comment:
1. It doesn't need to be a dict, use two variables to represent two numbers. 
2. I don't understand the "origin extra" well. Does it mean "origin" (created with VM) and "secondary" (not primary) nics? Please name it like secondary or something else, and specify what origin means in some comments.
3. How about other platforms than Azure? Can they use the same way to calculate the count?